resume.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+  <html>
+    <head>
+      <title> Rossy </title>
+        	<link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400&display=swap" rel="stylesheet">
+    		<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+      		<style>
+                  body {
+    padding-left: 10%;
+    padding-right: 10%;
+    margin-bottom: 10%;
+    background-color: #c3fbf4;
+}
+.Sticky {
+    width: 20vw;
+    height: 30vh;
+    position: fixed;
+    top: 0px;
+    z-index: 0;
+
+}
+.Main {
+    margin-top: 45vh;
+    background-color: #FF6978;
+    width: 80vw;
+    height: relative;
+    position: relative;
+    bottom: 30%;
+    z-index: 1;
+    opacity: 80%;
```

Comment:
I question this opacity choice since it actually impacts the readability of your content:
<img width="645" alt="Screen Shot 2020-10-04 at 3 26 57 PM" src="https://user-images.githubusercontent.com/2653980/95025203-1093ae00-0656-11eb-9c7d-ddeb638c1c07.png">

Instead of making the entire content have a [lowered opacity](https://mica-web.github.io/learn/css/opacity), consider using `rgba` or `hsla` to make only the background colors have opacity?